benchmark/parse_output.py

```diff
@@ -141,14 +142,110 @@ def parse_hgtector(input_f):
     return '\n'.join(hgts)
 
 
-def parse_output(hgt_results_fp, method):
+def parse_egid(input_f, genbank_fp):
+    """ Extract genes contained in GIs identified by EGID
+
+    Parameters
+    ----------
+    input_f: string
+        file descriptor for EGID output results (GI coordinates)
+    genbank_fp: string
+        file path to genome in GenBank format (containing gene coordinates)
+
+    Notes
+    -----
+    genbank_fp is the intermediate GenBank file generated by reformat_input.py,
+    in which multiple sequences are concantenated, instead of the original
+    GenBank file.
+
+    Returns
+    -------
+    output: string
+        gene names (protein_ids) separated by newline
+    """
+    genes = {}
+    gb = Sequence.read(genbank_fp, format='genbank')
+    for feature in gb.interval_metadata.features:
+        if feature['type_'] == 'CDS':
+            if 'protein_id' in feature:
+                protein_id = feature['protein_id'].replace('\"', '')
+                if protein_id not in genes:
+                    loc = gb.interval_metadata.features[feature]
+                    start = loc[0][0] + 1
```

Comment:
In scikit-bio, this number is the actual start location - 1. I am not sure why scikit-bio is coded like this (perhaps because computer languages count from 0 instead of 1). This won't affect other tools.